eth/fetcher/tx_fetcher.go

```diff
@@ -35,7 +35,7 @@ import (
 const (
 	// maxTxAnnounces is the maximum number of unique transaction a peer
 	// can announce in a short time.
-	maxTxAnnounces = 4096
+	maxTxAnnounces = 10000
```

Comment:
Are you sure they weren't using powers of 2 for a reason? Same for `maxTxRetrievals` below.